HelpSource/Classes/EnvironmentRedirect.schelp

```diff
@@ -15,7 +15,11 @@ subsection:: replacing Environment class methods
 
 EnvironmentRedirect implements some of the interface of link::Classes/Environment::
 
-method::push, pop, make, use, newFrom
+method::push, pop
```

Comment:
I'm unfamiliar with putting multiple methods on one line? Is there precedence for this? Also these can be explained or linked to the Environment class if that is appropriate.

Reply:
We can write multiple methods with the same arguments. If not, Sclang posts the following warning:
>Grouped methods [push, pop, make, use, newFrom] do not have the same argument signature.

To avoid this warning, this line has been divided.